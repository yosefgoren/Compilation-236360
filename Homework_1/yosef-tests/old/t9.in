// No newlines in strings!

int main () {
    print("Hello 
        World!");
    return 0;
}