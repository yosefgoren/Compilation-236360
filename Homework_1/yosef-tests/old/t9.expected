1 COMMENT //
3 INT int
3 ID main
3 LPAREN (
3 RPAREN )
3 LBRACE {
4 ID print
4 LPAREN (
Error unclosed string
